archon.sh

```diff
@@ -218,7 +224,17 @@ function chroot_stage {
 	echo							#
 	done							#
 	#########################################################
-	echo "$onomaxristi ALL=(ALL) ALL" >> /etc/sudoers
+	
+	## DELETE START ?
+	## echo "$onomaxristi ALL=(ALL) ALL" >> /etc/sudoers
+	## DELETE END ?
+
+	## ADDITION START ?
+	## Since new user belongs to wheel group now (via useradd command), 
+	## just uncomment the appropriate line  from the /etc/sudoers file
+	sed -i "s/# %wheel ALL=(ALL) ALL/%wheel ALL=(ALL) ALL/" $sudoers_file
+	## ADDITION END ?
```

Comment:
As mentioned before there is no need to use a variable for something used only once. Οπότε εδώ καλύτερα να χρησιμοποιείς απευθείας την /etc/sudoers